### ISSUE TITLE
Loading a Gothic1 SaveGame causes fetching some ZenKit.Vobs.Npc List properties to cause errors

### ISSUE THREAD

**Issue by JaXt0r** [GothicKit/ZenKitCS#13]
## Background
When loading a save game and accessing Vob data from NPCs (the ones far away), we get errors as the arrays aren't initialized.
The issue is tested and reproduced for elements which create a List on demand (_Missions_ and _HitChance_; Others might be affected as well but aren't tested.)

![image](https://github.com/user-attachments/assets/83ba3d70-422d-47db-aefe-b9d0adc236ef)

## Expected behaviour
Arrays of all properties should be initialized in named size.

## Example
```c#
var saveGameWorld = new SaveGame(...).LoadWorld(...);

foreach (var npc in SaveGameWorld.Npcs)
{
  var missions = npc.Missions; // Error: "ZkNpc_getMission() failed: index out of range"
}
```

## Technical details
Most probably, the issue appears as a Get of the List entries is assuming a fixed size which isn't initialized internally on ZenKit.

```c#
public class ZenKit.Vobs.Npc
{
  public List<int> Missions
  {
    get
    {
      List<int> missions = new List<int>();
      for (int i = 0; i < 8; ++i)
        missions.Add(this.GetMission(i));
      return missions;
    }
  }

  public int GetMission(int i) => Native.ZkNpc_getMission(this.Handle, (ulong) i);
}
```

**Comment by lmichaelis** [comment]
Yes indeed, looks like I copied the wrong sizes for the static arrays from the native header. f94d565 should fix it.

**Comment by JaXt0r** [comment]
It works marvellous. Thank you.
